ingestion/functions/parsing/brazil_distrito_federal/distrito_federal.py

```diff
@@ -0,0 +1,196 @@
+_STATE = "UF"
+_DATE_CONFIRMED = "Data Cadastro"
+_DATE_SYMPTOMS = "dataPrimeirosintomas"
+_DEATH = "Óbito"
+_LUNG = "Pneumopatia"
+_KIDNEY = "Nefropatia"
+_HEMATOLOGIC = "Doença Hematológica"
+_METABOLIC = "Distúrbios Metabólicos"
+_IMMUNOSUPPRESSED = "Imunopressão"
+_OBESITY = "Obesidade"
+_OTHERS = "Outros"
+_CARDIOVASCULAR = "Cardiovasculopatia"
+
+_COMORBIDITIES_MAP = {
+    "Pneumopatia": "lung disease",
+    "Nefropatia": "kidney disease",
+    "Distúrbios Metabólicos": "disease of metabolism",
+    "Obesidade": "obesity",
+    "Cardiovasculopatia": "cardiovascular system disease"
+}
+
+
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+    """
+    # Some date fields are empty
+    if raw_date:
+        # There is variation in how dates are reported
+        if "-" in raw_date:
+            date = datetime.strptime(raw_date, "%Y-%m-%d")
+        else:
+            date = datetime.strptime(raw_date, "%d/%m/%Y")
+        return date.strftime("%m/%d/%YZ")
+    else:
+        return None
+
+
+def convert_gender(raw_gender: str):
+    if raw_gender == "Masculino":
+        return "Male"
+    if raw_gender == "Feminino":
+        return "Female"
+
+
+def convert_events(date_confirmed, date_symptoms, death):
+    """There is no date of death"""
+    events = [
+        {
+            "name": "confirmed",
+            "dateRange": {
+                "start": convert_date(date_confirmed),
+                "end": convert_date(date_confirmed),
+            },
+        }
+    ]
+    if date_symptoms:
+        events.append(
+            {
+                "name": "onsetSymptoms",
+                "dateRange": {
+                    "start": convert_date(date_symptoms),
+                    "end": convert_date(date_symptoms),
+                },
+            }
+        )
+    if death == "Sim":
+        events.append(
+            {
+                "name": "outcome",
+                "value": "Death",
+            }
+        )
+    return events
+
+
+def convert_preexisting_conditions(lung: str, kidney: str, metabolic: str,
+                                   cardiovascular: str, obesity: str):
+    preexistingConditions = {}
+    if all((lung, kidney, metabolic, cardiovascular, obesity)) == "Sim":
+        return None
+
+    preexistingConditions["hasPreexistingConditions"] = True
+    commorbidities_list = []
+
+    if lung == "Sim":
+        commorbidities_list.append(_COMORBIDITIES_MAP["Pneumopatia"])
+    if kidney == "Sim":
+        commorbidities_list.append(_COMORBIDITIES_MAP["Nefropatia"])
+    if metabolic == "Sim":
+        commorbidities_list.append(_COMORBIDITIES_MAP["Distúrbios Metabólicos"])
+    if cardiovascular == "Sim":
+        commorbidities_list.append(_COMORBIDITIES_MAP["Cardiovasculopatia"])
+    if obesity == "Sim":
+        commorbidities_list.append(_COMORBIDITIES_MAP["Obesidade"])
+
+    preexistingConditions["values"] = commorbidities_list
```

Comment:
maybe check that there's anything in that array before setting it to avoid sending empty arrays for nothing?

```
if comorbidities:
  preexistingConditions["values"] = commorbidities